backtester/engine/live_test.go

```diff
@@ -65,21 +64,33 @@ func TestSetupLiveDataHandler(t *testing.T) {
 
 func TestStart(t *testing.T) {
 	t.Parallel()
-	dc := &dataChecker{
-		shutdown: make(chan bool),
-	}
-	err := dc.Start()
-	assert.NoError(t, err)
+	t.Run("successful start", func(t *testing.T) {
+		t.Parallel()
+		dc := &dataChecker{
+			shutdown: make(chan bool),
+		}
+		err := dc.Start()
+		assert.NoError(t, err)
```

Comment:
🚧 I think this needs to be a require
```suggestion
		require.NoError(t, err)
```

Reply:
Thanks, I agree it's better.